scijava/scijava-ops/src/test/java/org/scijava/param/JavadocParameterTest.java

```diff
@@ -110,6 +126,101 @@ public void testJavadocMethodIR() {
 		isSuitableGenericOpMethodInfo(info);
 	}
 	
+	@Test
+	public void testJavadocMethodI() {
+		Iterator<OpInfo> infos = ops.env().infos("test.javadoc.methodI").iterator();
+
+		OpInfo info = infos.next();
+		if (infos.hasNext()) {
+			Assert.fail("Multiple OpInfos with name \"test.javadoc.method\"");
+		}
+		isSuitableGenericOpMethodInfo(info);
+	}
+
+	/**
+	 * Tests javadoc scraping of mutable taglet
+	 * 
+	 * @mutable foo the i/o argument
+	 */
+	@OpMethod(names = "test.javadoc.methodInplaceI", type = Inplaces.Arity1.class)
+	public static void OpMethodInplaceI(List<String> foo) {
+		for (int i = 0; i < foo.size(); i++) {
+			foo.set(i, foo.get(i) + " foo");
+		}
+	}
+
+	/**
+	 * Tests javadoc scraping of mutable taglet
+	 * 
+	 * @dependency inplace the Op being wrapped
+	 * @mutable foo the i/o argument
+	 */
+	@OpMethod(names = "test.javadoc.methodDependency", type = Inplaces.Arity1.class)
+	public static void OpMethodInplaceI(@OpDependency(
+		name = "test.javadoc.methodInplaceI") Inplaces.Arity1<List<String>> inplace,
+		List<String> foo)
+	{
+		inplace.mutate(foo);
+	}
+
+	@Test
+	public void testJavadocMethodInplaceI() {
+		Iterator<OpInfo> infos = ops.env().infos("test.javadoc.methodInplaceI").iterator();
+
+		OpInfo info = infos.next();
+		if (infos.hasNext()) {
+			Assert.fail("Multiple OpInfos with name \"test.javadoc.method\"");
+		}
+
+		// assert input names
+		String[] inputNames = info.inputs().stream().map(m -> m.getKey()).toArray(
+			String[]::new);
+		Assert.assertArrayEquals(inputNames, new String[] { "foo" });
+
+		// assert input descriptions
+		String[] inputDescriptions = info.inputs().stream().map(m -> m
+			.getDescription()).toArray(String[]::new);
+		Assert.assertArrayEquals(inputDescriptions, new String[] {

```

Comment:
Actual and expected are reversed here, no?
I find it helps me get it right more often by declaring the expected explicitly on its own line, like this:
```java
var expected = {"the i/o argument"};
var actual = info.inputs().stream().map(m -> m.getDescription()).toArray(String[]::new);
Assert.assertArrayEquals(expected, actual);
```

Reply:
Fixed with deaa0497